_data/tasks.yml

```diff
@@ -23,4 +23,6 @@ toc:
   section:
   - title: Assigning Pods to Nodes
     path: /docs/tasks/administer-cluster/assign-pods-nodes/
+  - title: Using DNS Horizontal Autoscaling
```

Comment:
I'm not a docs guy, but this doesn't feel like a "task" to me.  Maybe "Configuring DNS to scale with you cluster", or maybe this is such a default thing it's not really a task at all?